admin/dt-prayer-campaigns.php

```diff
@@ -183,6 +186,13 @@ class="button"
 
                         ?>
                      </div>
+                    <div id="postbox-container-1" class="postbox-container">
```

Comment:
could we style this box to be inline with the box next to it below the buttons?
It would create a nicer flow for the eye

Reply:
@squigglybob  what do you mean?

Reply:
At the moment it's like this...
![image](https://user-images.githubusercontent.com/9816214/190373565-9bce7a52-e1f8-46f5-b1f7-7d67741306c0.png)

I'm just wondering if it would look nicer like this
![image](https://user-images.githubusercontent.com/9816214/190373775-5974c374-c3c3-4054-bbc1-ff66b7e97690.png)

Reply:
Oh. I aggree